tools/getrealdeps.rb

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env ruby
+# getrealdeps for Chromebrew version 1.2
+# Author: Satadru Pramanik (satmandu) satadru at gmail dot com
+require 'fileutils'
+
+# Exit quickly if an invalid package name is given.
+if ARGV[0].nil? || ARGV[0].empty? || ARGV[0].include?('#')
+  puts 'Getrealdeps checks for the dependencies of a package.'
+  puts 'Dependencies are added if the package file is missing them.'
+  puts 'Usage: getrealdeps.rb <packagename>'
+  exit 1
+end
+
+pkg = ARGV[0].chomp('.rb')
+
+CREW_PREFIX = ENV['CREW_PREFIX'] || `crew const CREW_PREFIX`.split('=')[1].chomp
```

Comment:
Would it not make more sense to include lib/const and draw the variables from there?

Reply:
This can be run independently of the build. We don't need all the `const.rb` infrastructure...

Reply:
We could implement the 3 or more rule.  If you have to use this method 3 or more times, include const.rb?